fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -61,19 +61,24 @@ private val ICON_SIZE = 24.dp
  * @param normalTabCount The total amount of normal browsing tabs currently open.
  * @param isInDebugMode True for debug variant or if secret menu is enabled for this session.
  * @param onTabPageIndicatorClicked Invoked when the user clicks on a tab page indicator.
+ * @param onExitSelectMode Invoked when the user clicks on exit select mode button from the
+ * multi select banner.
  */
+@Suppress("LongParameterList")
 @Composable
 fun TabsTrayBanner(
     selectMode: TabsTrayState.Mode,
     selectedPage: Page,
     normalTabCount: Int,
     isInDebugMode: Boolean,
     onTabPageIndicatorClicked: (Page) -> Unit,
+    onExitSelectMode: () -> Unit,
```

Comment:
nit: add the click verbiage to to make it clear that this lambda is a click response.
```suggestion
    onExitSelectModeClick: () -> Unit,
```